index.html

```diff
@@ -533,6 +584,63 @@
 
             </div>
         </div>
+
+        <div id="settings-modal" class="modal fade">
+            <div class="modal-dialog modal-lg">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal">&times;</button>
+                        <h4 class="modal-title">Settings</h4>
+                    </div>
+                    <div class="modal-body">
+                        <div class="split-container-left">
+                          <p>
+                            <b>
+                              Board and Pieces
+                            </b>
+                            <br/>
+                            <i>Changes to board and pieces will only take effect, once you watch or start a new game.</i><br/>
```

Comment:
Ah.. right they'd be off center. But I strongly feel that the user shouldn't have to reload/start new game to see effects. Sure, that'd involve more work, but I think dynamically changing is the right way to do it.

So instead of scaling, maybe we could replace the meshes with the scaled meshes while applying the original meshes' properties (like positions, while taking into account the size differences)